src/transformers/models/ibert/utils/quant_modules.py

```diff
@@ -0,0 +1,538 @@
+import torch
```

Comment:
Usually, we put all those functionalities in the `modeling_...` file no? Should we make an exception here because there is so much functionality? @sgugger @LysandreJik

Reply:
I merged the `utils/quant_modules.py` and the `utils/quant_utils.py` together and removed the `utils` directory.